src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -26,6 +26,10 @@ public partial class RunCommand
 
         private List<string> _args;
         private bool ShouldBuild => !NoBuild;
+        private bool HasQuietVerbosity =>
+            RestoreArgs.All(arg => !arg.StartsWith("-verbosity:") ||
```

Comment:
We should use StringComparison.Ordinal here.

Reply:
I'll fix.

Reply:
Pushed.